src/validation.cpp

```diff
@@ -1009,64 +1075,17 @@ static bool AcceptToMemoryPoolWorker(CTxMemPool& pool, CValidationState& state,
         // There is a similar check in CreateNewBlock() to prevent creating
         // invalid blocks (using TestBlockValidity), however allowing such
         // transactions into the mempool can be exploited as a DoS attack.
-        unsigned int currentBlockScriptVerifyFlags = GetBlockScriptFlags(chainActive.Tip());
-        if (!CheckInputsFromMempoolAndCache(tx, state, view, pool, currentBlockScriptVerifyFlags, true, txdata)) {
+        if (!CheckInputsFromMempoolAndCache(tx, state, view, pool, SCRIPT_VERIFY_NONE, true, txdata)) {
```

Comment:
Why did you remove `currentBlockScriptVerifyFlags`?

Reply:
It looked redundant because there are no real flags now. I'm undoing this as you advice.